number_trivia_app/test/features/number_trivia/data/models/number_trivia_model_test.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:convert';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:number_trivia_app/features/number_trivia/data/models/number_trivia_model.dart';
+import 'package:number_trivia_app/features/number_trivia/domain/entities/number_trivia.dart';
+
+import '../../../../fixtures/fixture_reader.dart';
+
+void main() {
+  final tNumberTriviaModel = NumberTriviaModel(number: 1, text: 'Test Text');
```

Comment:
Should you add the named parameters on different lines?

Reply:
We discussed the topic offline